omas/machine_mappings/d3d.py

```diff
@@ -592,7 +592,10 @@ def ec_launcher_active_hardware(ods, pulse):
         beam['launching_position.phi'] = phi * np.ones(ntime)
 
         beam['frequency.time'] = np.atleast_1d(0)
-        beam['frequency.data'] = np.atleast_1d(systems[f'FREQUENCY_{system_no}'])
+        if isinstance(systems[f'FREQUENCY_{system_no}'], Exception):
+            beam['frequency.data'] = beam['frequency.time'] * 0.0 + 110e9 # old shots did not record the frequency, since they were all at 110 GHz
```

Comment:
For clarity
```suggestion
            beam['frequency.data'] = np.ones(beam['frequency.time'].shape) * 110e9 # old shots did not record the frequency, since they were all at 110 GHz
```